nginx.py

```diff
@@ -1,6 +1,28 @@
 from fabric.api import *
+import fabric.contrib.files
+import puppet
+
+maintenance_config = '/etc/nginx/includes/maintenance.conf'
```

Comment:
Those would ideally be constants also, like in [campaigns.py](https://github.com/alphagov/fabric-scripts/blob/5d24692cf8466de3fbdd5a46bd0218c8d3cc63fd/campaigns.py#L15-L16).